pagina de links

elizeumagalhaes opened the pull request.

viktormarinho commented:
oq vc ta fazendo
?

abehidek commented:
então, aparentemente, eu coloquei o commit da sidebar na branch `dev`, mas a PR do @elizeumagalhaes contém o commit da sidebar, sendo que ele foi deletado, estou vendo uma maneira de dar merge na pr sem o commit da sidebar

abehidek committed:
fix(pr/elizeumagalhaes): delete feat sidebar commit

Also:
- feat(config.ts): Add utils/links.md to sidebar
- add: frontmatter info in links.md (layout, title and description)
- delete: old package-lock.json

Co-authored-by: Elizeu Magalhães <elizeucontato@hotmail.com>

abehidek commented:
Tivemos que refazer o commit, então os commits anteriores, mas basicamente o que foi refeito:

- Excluir: package-lock.json  
Para evitar conflitos entre os gerenciadores de pacotes (npm e pnpm), o arquivo `package-lock.json` (criado automaticamente pelo `npm`) tem que ser excluído, pois estamos utilizando o arquivo `pnpm-lock.yaml` no lugar.
Todos os projetos estão utilizando o `pnpm` (você pode instalar na sua máquina seguindo o passo a passo: https://pnpm.io/pt/installation.
- Excluir: commit `feat(Sidebar): start rendering function using created three structure`
Esse commit é os testes relacionados que foram movidos para a [branch dev](https://github.com/tcc-ets/docs/tree/dev), mas a pull request reintroduz esse commit, para evitar o mesmo erro de antes, [esse commit foi deletado pois ele já foi movido para outra branch](https://github.com/tcc-ets/docs/commit/6c56c812cbc3206b60cbab9aa7e20388deecd725).
- Adicionado: `utills/links.md` para a barra lateral
Para a página de links ser adicionada na barra lateral, foi necessário adicionar ela no arquivo `src/config.ts`, mas isso provavelmente mudará no futuro, onde só precisará adicionar o arquivo markdown com alguns metadados (vide frontmatter) e ele automaticamente já coloca na barra lateral.
- Adicionado título e descrição e layout da página `utills/links.md` através dos metadados do markdown (frontmatter) e removido título da página em markdown.

viktormarinho reviewed (approved):
okay, LGTM!
